src/css/_mediaQueries.scss

```diff
@@ -0,0 +1,70 @@
+@media screen and (max-width: 550px) {
+
+  .main-container {
+    grid-template-columns: repeat(1, 1fr);
+    grid-template-rows: repeat(6, 1fr);
+  }
+
+  .header {
+    grid-column: 1 / 2;
+    grid-row: 1 / 2;
+    text-align: center;
+  }
+
+  .user-dash {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+  }
+
+  .activity-column {
+    grid-column: 1 / 2;
+    grid-row: 3 / 4;
+  }
+
+  .main-column-hydration {
+    grid-column: 1 / 2;
+    grid-row: 4 / 5;
+  }
+
+  .main-column-sleep {
+    grid-column: 1 / 2;
+    grid-row: 5 / 6;
+  }
+
+  .sidebar-header-challengeImage {
+    width: 40%;
+  }
+}
+
+@media screen and (max-width: 800px) and (min-width: 700px) {
```

Comment:
It is doing a weird thing where it is going to 2 columns, then back to 3 columns, before going back to 2 columns as the screen gets smaller. Perhaps try out something like `(max-width: 1024px) and (min-width: 550px)`

Reply:
got it! thanks!